lib/src/ui/pages/profile/auth/auth_text_form_field.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:allthenews/src/di/injector.dart';
+import 'package:allthenews/src/ui/pages/profile/auth/authorization_notifier.dart';
+import 'package:flutter/material.dart';
+
+abstract class _Constants {
+  static const fontFamily = 'Chomsky';
+  static const widgetPadding = 15.0;
+  static const borderRadius = 5.0;
+  static const focusedBorderSideWidth = 3.0;
+}
+
+class AuthTextFormField extends StatelessWidget {
+  final AuthorizationNotifier _authorizationNotifier = inject<AuthorizationNotifier>();
+  final TextEditingController textController;
+  final String hint;
+  final bool obscureText;
+
+  AuthTextFormField({
+    this.textController,
+    this.hint,
+    this.obscureText = false,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(_Constants.widgetPadding),
+      child: TextFormField(
+        autovalidateMode: AutovalidateMode.onUserInteraction,
+        controller: textController,
+        obscureText: obscureText,
+        decoration: InputDecoration(
+          labelText: hint,
```

Comment:
jakoś dziwnie wygląda jak do labelText jest przypisywany hint, trudno mi to wytlumaczyć, ale jakby było odwrotnie to jakoś mniej dziwnie by mi to wyglądało, może dlatego, ze każdy label pasuje jako hint ale nie każdy hint pasuje jako label 🤔 - nie wiem czy słusznie, teraz przyszło mi do głowy :D